spec/emerald/rubify_spec.rb

```diff
@@ -0,0 +1,131 @@
+require "spec_helper"
+require "emerald/rubify"
+require "emerald/atom"
+require "emerald/string"
+require "emerald/number"
+require "emerald/list"
+
+RSpec.describe Emerald::Rubify do
+
+  it "compiles an empty source" do
+    compiled_code = Emerald::Rubify.new([]).rubify
+    expect(compiled_code).to eq('')
+  end
+
+  describe "code generation from atoms" do
+    it "generates code from a single atom" do
+      compiled_code = Emerald::Rubify.new([Emerald::Atom.new("hello-world")]).rubify
+      expect(compiled_code).to eq('hello-world')
```

Comment:
Yee, but in Ruby it's not valid.

```
louis ~ $ irb
irb(main):001:0> hello-world
NameError: undefined local variable or method `hello' for main:Object
Did you mean?  help
	from (irb):1
	from /Users/louispilfold/.rbenv/versions/2.3.0/bin/irb:11:in `<main>'
```

Bare atoms could compile to symbols, I think that makes sense, so `:"hello-world"`.